Makefile.common

```diff
@@ -72,14 +72,13 @@ endif
 PROMU_VERSION ?= 0.3.0
 PROMU_URL     := https://github.com/prometheus/promu/releases/download/v$(PROMU_VERSION)/promu-$(PROMU_VERSION).$(GO_BUILD_PLATFORM).tar.gz
 
-STATICCHECK :=
+GOLANGCI_LINT :=
+GOLANGCI_LINT_VERSION ?= v1.16.0
 # staticcheck only supports linux, freebsd, darwin and windows platforms on i386/amd64
 # windows isn't included here because of the path separator being different.
 ifeq ($(GOHOSTOS),$(filter $(GOHOSTOS),linux freebsd darwin))
 	ifeq ($(GOHOSTARCH),$(filter $(GOHOSTARCH),amd64 i386))
```

Comment:
are these checks still needed?

Reply:
I've removed freebsd since it isn't supported by golangci-lint.